<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nine</class>
 <widget class="QMainWindow" name="nine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="_visit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>320</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>计时：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>410</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>得分：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>570</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>AI(只支持3*3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="_score">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>410</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>320</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>难度选择</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
    </widget>
    <addaction name="action"/>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>初级</string>
   </property>
  </action>
  <action name="action_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中级</string>
   </property>
  </action>
  <action name="action_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>高级</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header location="global">myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
